packages/merge-styles/api/merge-styles.api.ts

```diff
@@ -161,8 +161,8 @@ interface IRawStyle extends IRawFontStyle {
   maskBorderWidth?: ICSSRule | ICSSPixelUnitRule;
   maskClip?: ICSSRule | string;
   maskOrigin?: ICSSRule | string;
-  maxFontSize?: ICSSRule | string;
-  maxHeight?: ICSSRule | string;
+  maxFontSize?: ICSSRule | ICSSPixelUnitRule;

```

Comment:
I changed maxFontSize and maxHeight because components have these props set as numbers, so with ICSSPixelUnitRule it can be either a number and string (the same we do for other length css rules). Just wanted to verify it is all right with you guys and there are no concerns?